data-prepper-plugins/cloudwatch-logs/src/test/java/org/opensearch/dataprepper/plugins/sink/configuration/CwlSinkConfigTest.java

```diff
@@ -0,0 +1,30 @@
+package org.opensearch.dataprepper.plugins.sink.configuration;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.opensearch.dataprepper.plugins.sink.config.CwlSinkConfig;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+public class CwlSinkConfigTest {
+    @Test
+    void check_null_auth_config_test() {
+        assertThat(new CwlSinkConfig().getAwsConfig(), equalTo(null));
```

Comment:
You can use this [ReflectivelySetField](https://github.com/opensearch-project/data-prepper/blob/6648d9fea792cfca889c69f62ce50af6ee12392a/data-prepper-test-common/src/main/java/org/opensearch/dataprepper/test/helper/ReflectivelySetField.java) class to configure the options and verify if they're being set correctly.

Reply:
Thank you for letting me in on this. I will re-write the tests for the configs to match these.